single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -785,20 +785,20 @@ protected void transformArrow(List<Point> points, ComponentSize componentSize, d
             double cosAngle = dx / distancePoints;
             double sinAngle = dy / distancePoints;
 
-            // If not enough space to have layoutParameters.getArrowDistance() at both sides of the 2 arrows,
-            // we compute the distance between feeder anchor and first arrow so that the two arrows are centered.
-            double distFeederAnchorToFirstArrowCenter =
-                distancePoints >= 2 * layoutParameters.getArrowDistance() + 2 * componentSize.getHeight()
-                    ? layoutParameters.getArrowDistance()
+            // If not enough space to have layoutParameters.getArrowDistance() at both sides of the 2 feeder infos,
+            // we compute the distance between feeder anchor and first feeder info so that the two feeder infos are centered.
+            double distFeederAnchorToFirstFeederInfoCenter =
+                distancePoints >= 2 * layoutParameters.getFeederInfosOuterMargin() + 2 * componentSize.getHeight()
+                    ? layoutParameters.getFeederInfosOuterMargin()
                     : (distancePoints - 2 * componentSize.getHeight()) / 2;
-            double x = pointA.getX() + cosAngle * (distFeederAnchorToFirstArrowCenter + shift);
-            double y = pointA.getY() + sinAngle * (distFeederAnchorToFirstArrowCenter + shift);
+            double x = pointA.getX() + cosAngle * (distFeederAnchorToFirstFeederInfoCenter + shift);
+            double y = pointA.getY() + sinAngle * (distFeederAnchorToFirstFeederInfoCenter + shift);
 
```

Comment:
Could you delete this code? I think there's no use to keep it now that the user is responsible for giving enough space for the feederInfos

Reply:
I've change as following : 
double distFeederAnchorToFirstFeederInfoCenter = layoutParameters.getFeederInfosOuterMargin()